Dockerfile

```diff
@@ -16,6 +16,7 @@ RUN pip install \
 COPY geospaas /tmp/geospaas
 COPY setup.py /tmp/
```

Comment:
You need to copy the `MANIFEST.in` file, otherwise it won't be used:
```
COPY setup.py MANIFEST.in /tmp/
```

Reply:
Done